nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -36,6 +37,52 @@
 namespace nebula::drivers
 {
 
+struct HesaiDecodeFilteredInfo
+{
+  uint16_t distance_counter = 0;
+  uint16_t fov_counter = 0;
+  uint16_t timestamp_counter = 0;
+  float distance_start = 0;
+  float distance_end = 0;
+  float raw_azimuth_start = 0;
+  float raw_azimuth_end = 0;
+  std::uint32_t packet_timestamp_start = 0;
+  std::uint32_t packet_timestamp_end = 0;
+  NebulaPointCloud point_azimuth_start;
+  NebulaPointCloud point_azimuth_end;
+  NebulaPointCloud point_timestamp_start;
+  NebulaPointCloud point_timestamp_end;
```

Comment:
Please change these from `NebulaPointCloud` to their respective data types (e.g. `float` for azimuth, `uint64_t` for timestamp.